website/content/docs/connect/gateways/index.mdx

```diff
@@ -35,10 +35,42 @@ Mesh gateways enable the following scenarios:
 
 -> **Mesh gateway tutorial**: Follow the [mesh gateway tutorial](/consul/tutorials/developer-mesh/service-mesh-gateways) to learn concepts associated with mesh gateways.
 
+## API Gateways
+
+-> **1.15.0+:** This feature is available in Consul versions 1.8.0 and newer.
```

Comment:
```suggestion
-> **1.15.0+:** This feature is available in Consul versions 1.8.0 and newer.
```
Which version number are we claiming? According to the release notes, the first release was 1.11.